MapboxMaps.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |m|
 
-  maps_version = '10.8.0-rc.1'
+  maps_version = '10.8.0'
```

Comment:
Along with this version in `MapboxMaps.json` and `CFBundleVersion` needs to be updated, did you run `scripts/release/update-version.sh` script?

Reply:
Done, thank you!